website_product_filters/controllers/main.py

```diff
@@ -1,4 +1,5 @@
 # coding: utf-8
+# pylint: disable=expression-not-assigned
```

Comment:
You are disabling such element in all file I prefer disable it into the specific line.